### PR TITLE
Add landing message. #1

### PR DESCRIPTION
**Pull request by marisudris**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Welcome to Potionfy</title>
```
Comment (harlequiin) [inline index.html:6]: Could you remove the `Enjoy your stay.` string and make the `Welcome to Potionfy!` string red?

Comment (marisudris) [reply index.html:6]: Sure can do!

**Review by harlequiin — CHANGES_REQUESTED** [review]
Please, change the code as per my comments on the review.

**Review by harlequiin — APPROVED** [review]
Well done, Maris!

**Pull request merged**